src/org/opensolaris/opengrok/search/Results.java

```diff
@@ -171,7 +175,15 @@ public static void prettyPrint(Writer out, SearchHelper sh, int start,
                 out.write(rpathE);
                 out.write("\">");
                 out.write(rpath.substring(rpath.lastIndexOf('/') + 1)); // htmlize ???
-                out.write("</a></td><td><tt class=\"con\">");
+                out.write("</a>");
+                try {
+                    // insert last edited date if possible
+                    df = DateFormat.getDateTimeInstance(DateFormat.SHORT, DateFormat.SHORT);
+                    out.write("<small class=\"edited\">(last edited ");
+                    out.write(df.format(DateTools.stringToDate(doc.get("date"))));
```

Comment:
I have some doubts you will have unclosed <small tag if there will be an exception ...